.github/workflows/pull_request_cleanup_tags_and_releases.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Remove unused releases and tags
-        uses: Broadshield/github-action-cleanup-releases-and-tags@main
+        uses: Broadshield/github-action-cleanup-releases-and-tags@v1.0.0
```

Comment:
@Jamie-BitFlight this action looks like not maintained, why do we need to clean tags?

Reply:
It lets you manually enter a regex tag pattern to remove tags or releases that are unneeded.
I started using it because I was testing out a bunch of different build and tagging systems before i settled on 'semantic-versioning' which does what II am after. 
It probably can be removed if you want to do a pull request to delete iit?